File: Java/FacadePattern/src/cluster_of_classes/Toyota.java
package cluster_of_classes;

/**
 * Author: Utkarsh Dayal @ descifrador17
 * Date: 04-06-2021
 * Time: 03:07 AM
 */
public class Toyota implements Car{
    @Override
    public void model() {
        System.out.println("Toyota Supra");
    }

    @Override
    public void price() {
        System.out.println("$40000");
    }
}

File: Kotlin/FacadePattern/src/cluster_of_classes/Toyota.kt
package cluster_of_classes

/**
 *
 *   Author: Utkarsh Dayal @ descifrador17
 *   Date: 04-06-2021
 *   Time: 03:23 AM
 *
 */
class Toyota: Car {
    override fun model() {
        println("Toyota Supra")
    }

    override fun price() {
        println("$40000")
    }
}